docs/install_with_docker.md

```diff
@@ -0,0 +1,47 @@
+# Install BladeDISC with Docker
+
+[Docker](https://www.docker.com/) is a light container system, it helps
+users to package software and isolate BladeDISC runtime environment from
+the rest of the system. BladeDISC CI system released BladeDISC with different
+tag on [Docker Hub](https://hub.docker.com/repository/docker/bladedisc/bladedisc/tags?page=1&ordering=last_updated)
+repository.
+
+[Nvidia Container Toolkit](https://github.com/NVIDIA/nvidia-docker/blob/master/README.md#quickstart)
+is an easy way to use GPU on Linux, please make sure you have installed it
+on your host.
+
+## Download a BladeDISC Docker Image
+
+BladeDISC released TensorFlow and PyTorch wrappers in separate Docker images
```

Comment:
nit: TensorFlow and PyTorch wrappers -> TensorFlow and PyTorch frontend packages?